projects/1.19/assets/dimensional-pockets-ii/dimensionalpocketsii/lang/zh_cn.json

```
@@ -0,0 +1,1072 @@
+	
+	"dimensionalpocketsii.direction.generic.pre": "成功移动：",
+	"dimensionalpocketsii.direction.generic.mid": "到：",
+	
+	"dimensionalpocketsii.pocket.shifter_core.target_unknown_custom": "无法移动至正确的目标处",
+	"dimensionalpocketsii.pocket.shifter_core.target_unknown": "无法移动至正确的目标处。放弃世界生成",
+	"dimensionalpocketsii.pocket.shifter_core.direction_unknown": "移动已被取消。移向[[未知]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.shifter_core.dimension_null": "移动已被取消。次元为[[空]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.shifter_core.server_world_null": "移动已被取消。服务器世界为[[空]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.shifter_core.same_dimension": "移动已被取消。你已经在目标次元里面了！",
+	
+	"dimensionalpocketsii.keybind.category": "次元口袋 II",
+	"dimensionalpocketsii.keybind.suit_screen": "鞘翅胸甲显示屏键",
+	"dimensionalpocketsii.keybind.suit_shift": "鞘翅胸甲移动键",
+	"dimensionalpocketsii.keybind.suit_mode_change": "鞘翅胸甲设置键",
+	"dimensionalpocketsii.keybind.suit_ender_chest": "鞘翅胸甲末影箱键",
+	
+	"dimensionalpocketsii.item.message.shifter.linked": "链接到口袋的次元变速箱：",
+	"dimensionalpocketsii.item.message.shifter.not_linked": "你还没有将变速箱链接到你的口袋呢！",
+	"dimensionalpocketsii.item.message.shifter.not_owner": "你不能将变速箱链接到该口袋，只因你并非该口袋的所有者",
+	"dimensionalpocketsii.item.message.shifter.mode_change": "模式更改：",
+	"dimensionalpocketsii.item.message.shifter.mode_change_prefix": "传送到方块：",
+	"dimensionalpocketsii.item.message.shifter.mode_change_true": "true",
+	"dimensionalpocketsii.item.message.shifter.mode_change_false": "false",
+	"dimensionalpocketsii.item.message.shifter.mode_change_suffix": "]",
+	"dimensionalpocketsii.item.message.shifter.no_energy": "你的次元变速箱能量不足！",
+	
+	"dimensionalpocketsii.item.message.ejector.success": "已成功从该次元中弹出：",
+	
+	"dimensionalpocketsii.item.info.tool_charge": "充电：放置在次元充电模块处",
+	"dimensionalpocketsii.item.info.tool_usage": "每次成功进行将使用2500 FE 的能量",
+	"dimensionalpocketsii.item.info.tool_usage_enhanced": "每次成功进行将使用2500 FE 的能量",
+	"dimensionalpocketsii.item.info.tool_energy": "最大储能：4,000,000；最大输入/输出：50,000",
+	"dimensionalpocketsii.item.info.tool_energy_enhanced": "最大储能：5,000,000；最大输入/输出：100,000",
+	"dimensionalpocketsii.item.info.tool_limitation": "能量耗尽时将不会正常工作",
+	
+	"dimensionalpocketsii.item.info.sword": "一把使用 FE 能量且强有力的剑",
+	"dimensionalpocketsii.item.info.pickaxe": "一把使用 FE 能量且强有力的镐",
+	"dimensionalpocketsii.item.info.shovel": "一把使用 FE 能量且强有力的锹",
+	"dimensionalpocketsii.item.info.axe": "一把使用 FE 能量且强有力的斧",
+	"dimensionalpocketsii.item.info.hoe": "一把使用 FE 能量且强有力的锄",
+	
+	"dimensionalpocketsii.item.info.sword_enhanced": "一把使用 FE 能量且无敌的剑",
+	"dimensionalpocketsii.item.info.pickaxe_enhanced": "一把使用 FE 能量且无敌的镐",
+	"dimensionalpocketsii.item.info.shovel_enhanced": "一把使用 FE 能量且无敌的锹",
+	"dimensionalpocketsii.item.info.axe_enhanced": "一把使用 FE 能量且无敌的斧",
+	"dimensionalpocketsii.item.info.hoe_enhanced": "一把使用 FE 能量且无敌的锄",
+	
+	"dimensionalpocketsii.item.info.trident": "一把使用 FE 能量且强有力的三叉戟",
+	"dimensionalpocketsii.item.info.trident_enhanced": "一把使用 FE 能量且无敌的三叉戟",
```

```suggestion
	"dimensionalpocketsii.item.info.sword": "一把使用FE能量且强有力的剑",
	"dimensionalpocketsii.item.info.pickaxe": "一把使用FE能量且强有力的镐",
	"dimensionalpocketsii.item.info.shovel": "一把使用FE能量且强有力的锹",
	"dimensionalpocketsii.item.info.axe": "一把使用FE能量且强有力的斧",
	"dimensionalpocketsii.item.info.hoe": "一把使用FE能量且强有力的锄",
	
	"dimensionalpocketsii.item.info.sword_enhanced": "一把使用FE能量且无敌的剑",
	"dimensionalpocketsii.item.info.pickaxe_enhanced": "一把使用FE能量且无敌的镐",
	"dimensionalpocketsii.item.info.shovel_enhanced": "一把使用FE能量且无敌的锹",
	"dimensionalpocketsii.item.info.axe_enhanced": "一把使用FE能量且无敌的斧",
	"dimensionalpocketsii.item.info.hoe_enhanced": "一把使用FE能量且无敌的锄",
	
	"dimensionalpocketsii.item.info.trident": "一把使用FE能量且强有力的三叉戟",
	"dimensionalpocketsii.item.info.trident_enhanced": "一把使用FE能量且无敌的三叉戟",
```
这个真的是指无敌吗
原文`ultimate`